### ISSUE TITLE
A11y_SqlDatabaseProjectsExtension_CreateNewDatabaseProject_AzureSQLdatabase_ScreenReader: ORCA screen reader is announcing invalid entry when we navigate to "Name" required field.

### ISSUE THREAD

**Issue by PoojaNamde** [microsoft/azuredatastudio#23551]
### Actual Result:
ORCA screen reader is announcing invalid entry when we navigate to "Name" required field.

### Expected Result:
ORCA Screen reader should not announce invalid entry when we navigate to "Name" required field.

### Git hub Tags:
#A11yTCS; #A11yMAS; #A11ySev2; #DesktopApp; #Benchmark; #ORCA; #WCAG1.3.1; #FTP; #SH-SQL Azure Data Studio-Linux-Jul23; #SQL Azure Data Studio; #Linux; #Element:Editfield;

### Environment Details:
Application Name: Sql Database Projects Extension
Operating System: Ubuntu 20.04.2 LTS
Screen Reader: ORCA

### Repro Steps:
1. Launch Azure Data Studio application.
2. TAB to 'Extension' and search for “SQL Database Projects” and install it.
3. TAB to "Database project" and press ENTER key.
4. Start ORCA screen reader. TAB till “Create New” button and press ENTER key.
5. “Create New Database Project” window will open(default type will set to Azure SQL database).
6. TAB to "Name" required field and observe the issue.

### User Impact:
The screen reader users will get incorrect information and will get confused about why the screen reader is announcing field as invalid entry.

### Recommendations:
Refer [Repository of bug fixes code snippets](https://nam06.safelinks.protection.outlook.com/?url=https:%2f%2faka.ms%2faccsnippet&data=05%7c01%7cv-samhithaks%40microsoft.com%7c535b4a834ef843acdfb508da53769f91%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637914066217264155%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c3000%7c%7c%7c&sdata=xoIfLB9Iq6HVIPyUc42rS%2BlVQCt9m%2BaUokp32Kaz1nE%3D&reserved=0).

[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier.

**"Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility or the Divisional Driver"**

**Comment by kisantia** [comment]
@PoojaNamde a similar issue https://github.com/microsoft/azuredatastudio/issues/22221 of announcing invalid entry for a required input is a sev3. Can this issue also be downgraded to a sev3?

**Comment by PoojaNamde** [comment]
This is already in Sev3.

**Comment by kisantia** [comment]
ah I didn't notice that get edited the severity changed to sev3 on July 12. Thanks!
